lib/devices/device.js

```diff
@@ -34,7 +35,7 @@ Device.prototype.configure = function (args, caps) {
     // use a custom tmp dir to avoid loosing data and app
     // when computer is restarted
     this.args.tmpDir = process.env.APPIUM_TMP_DIR ||
-      (helpers.isWindows() ? process.env.TEMP : "/tmp");
+      (isWindows() ? process.env.TEMP : "/tmp");
```

Comment:
Is there any reason that this dir already exists and may contain data from previous sessions?  Or that some other application is using a dir of the same name?  Might be nice to use a UUID rather than "/tmp".  Not a huge deal, but if we're changing this already.